### PR TITLE
[15.0][MIG]analytic_partner: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

**Comment by bizzappdev** [comment]
ready for review

**Review by stefan-tecnativa — APPROVED** [review]

File: analytic_partner/README.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to **Setting > Users > Users** and check you have as at least "Adviser"
```
Comment (pedrobaeza) [inline analytic_partner/README.rst:44]: These instructions are obsolete.

**Comment by pedrobaeza** [comment]
Superseded by #573 respecting commit attribution.

**Pull request closed without merging**